torch/onnx/symbolic_opset9.py

```diff
@@ -1493,6 +1493,17 @@ def scalar_tensor(g, scalar, dtype, *options):
     scalar = g.op("Cast", scalar, to_i=sym_help.scalar_type_to_onnx[dtype])
     return scalar
 
+def tensor(g, data, dtype=None, device=None, requires_grad=False):
+    value = sym_help._maybe_get_const(data, 't')
+    if dtype is None:
+        dtype = 6  # float
+    else:
+        if sym_help._is_packed_list(data):
+            unsqueezed = [g.op("Unsqueeze", t, axes_i=[0]) for t in sym_help._unpack_list(data)]
```

Comment:
Since elements in the list could be either scalar tensors or 1D tensors, we might not want to unsqueeze all of those.
Maybe try to construct a single element 1D tensor and use view_as() to reshape the nodes accordingly.

Reply:
Made necessary changes